README.md

```diff
@@ -1 +1,7 @@
-# go-internship
\ No newline at end of file
+# Assignments for Golang internship
+## Task 1 - Reverse a string
+1. Implement a function that reverses a string. Cover your code with unit tests, use TDD
+   approach. Create a package for all the source files. Put your package in a module.
+   Publish your solution to a remote repo. Release the module under version 1.0
+2. Implement a function that capitalizes a string
+
```

Comment:
Let's describe this module from the user's perspective? What problems does it solve, what is it capable of?